netstandard/ref/System.Xml.Serialization.cs

```diff
@@ -56,6 +57,10 @@ public partial interface IXmlTextParser
         bool Normalized { get; set; }
         System.Xml.WhitespaceHandling WhitespaceHandling { get; set; }
     }
+    public abstract partial class SchemaImporter
+    {
+        internal SchemaImporter() { }
```

Comment:
Why is there an `internal` member in a reference assembly? Better yet, since the type is abstract, why does it need any ctor?